react/components/Table/util/getInitialHiddenFieldsFromSchema.js

```diff
@@ -0,0 +1,9 @@
+export default schema => {
+  const hiddenFields = []
+  Object.keys(schema.properties).forEach(key => {
```

Comment:
Isn't this funciton just a filter? :thinking:

Reply:
Could be, just extracted the behavior that was on `Table`. Do you think the change is needed?

Reply:
done